app/presenters/areas_presenter.rb

```diff
@@ -35,7 +35,8 @@ def area_attrs(area)
       "slug" => area["name"].parameterize,
       "name" => area["name"],
       "country_name" => area["country_name"],
-      "type" => area["type"]
+      "type" => area["type"],
+      "gss"  => area.fetch("codes", {})["gss"],
```

Comment:
We might also want to expose the SNAC code in future to help with the migration from SNAC to GSS, so nesting `gss` under `codes` as it is in the MapIt response would make sense for that.

Nesting could also give us more flexibility in the future, in case we want to switch to calling MapIt directly from other applications. I'm not sure how much benefit there is to going through Imminence for these area lookups, since API adapters could do the presentation and the only substantive modifications that Imminence makes to the response are:

- slugifying the area names: we're moving away from using this at the moment
- [adding a region for England](https://github.com/alphagov/imminence/blob/26f6c9e5969a9e09bd24d6e2e4ebfe55dba1d997/lib/mapit_api.rb#L49) into the results for the EUR area type: this currently [doesn't work as originally intended](https://trello.com/c/xC6xdi6O) and should probably be removed
- [returning 404](https://github.com/alphagov/imminence/blob/26f6c9e5969a9e09bd24d6e2e4ebfe55dba1d997/lib/mapit_api.rb#L65) for all postcode lookup requests which have an empty response: I think this, in combination with the way that the [API adapter](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/mapit.rb#L9-L11) works, is obscuring MapIt's distinction between valid-looking but missing postcodes (`404`) and input which doesn't look like a postcode at all (`400`). That distinction could be useful to the frontend app to display different error messages to the user.